crypto/fipsmodule/aes/aes.c

```diff
@@ -323,41 +315,41 @@ int GFp_aes_c_set_encrypt_key(const uint8_t *key, unsigned bits,
       rk[6] = rk[2] ^ rk[5];
       rk[7] = rk[3] ^ rk[6];
       if (++i == 10) {
-        return 0;
+        return;
       }
       rk += 4;
     }
   }
+
+  // bits == 256
```

Comment:
This should be `assert(bits == 256);`. That is, we prefer checked (at debug time at least) assertions to unchecked assertions.